src/omnisharp/server.ts

```diff
@@ -304,6 +304,15 @@ export class OmniSharpServer {
             '--loglevel', options.loggingLevel
         ];
 
+        if (!options.razorDisabled) {
+            const razorPluginPath = path.join(
+                utils.getExtensionPath(),
```

Comment:
server takes an extension path in the constructor. We should use that here

Reply:
Good plan. Done.